Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py

```diff
@@ -1,14 +1,14 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
-from PyPDF2 import PdfFileReader, PdfFileWriter
+from PyPDF2 import PdfReader, PdfFileWriter
 
 
 def unlock_pdf(entry_id):
     res = demisto.getFilePath(entry_id)
     origin_path = res['path']
     output_name = "UNLOCKED_" + res['name']
 
-    input1 = PdfFileReader(open(origin_path, "rb"))
+    input1 = PdfReader(open(origin_path, "rb"))
```

Comment:
how will this pass on the current 8.1?

Reply:
its deprecated, not removed. Why did this fail the build before?

Reply:
https://github.com/py-pdf/pypdf/blob/main/pypdf/_reader.py#L2077 which leads to https://github.com/py-pdf/pypdf/blob/main/pypdf/_utils.py#L384 (Exception).

Reply:
by the way it seems like it complains about more classes/methods which were deprecated, changed it according to their recommendations?

Reply:
you mean about deprecated integrations? They shouldnt be changed

Reply:
i mean there are several more places which needs to be updated in the PYpdf2 module as well (as you see in the PR)